Source/AwsCommonRuntimeKit/http/HttpClientConnectionManager.swift

```diff
@@ -16,30 +17,38 @@ public class HttpClientConnectionManager {
         self.options = options
         self.allocator = allocator
         let shutDownPtr: UnsafeMutablePointer<ShutDownCallbackOptions>? = fromOptionalPointer(ptr: options.shutDownOptions)
+        //Todo: add new http2 options as properties in HttpClientConnectionOptions
         var mgrOptions = aws_http_connection_manager_options(bootstrap: options.clientBootstrap.rawValue,
-                                                             initial_window_size: options.initialWindowSize,
-                                                             socket_options: options.socketOptions.rawValue,
-                                                             tls_connection_options: options.tlsOptions?.rawValue,
-                                                             monitoring_options: options.monitoringOptions?.rawValue,
-                                                             host: options.hostName.awsByteCursor,
-                                                             port: options.port,
-                                                             proxy_options: options.proxyOptions?.rawValue,
-                                                             proxy_ev_settings: options.proxyEnvSettings?.rawValue,
-                                                             max_connections: options.maxConnections,
-                                                             shutdown_complete_user_data: shutDownPtr,
-                                                             shutdown_complete_callback: { (userData) in
-                                                                guard let userData = userData else {
-                                                                    return
-                                                                }
+                initial_window_size: options.initialWindowSize,
+                socket_options: options.socketOptions.rawValue,
+                tls_connection_options: options.tlsOptions?.rawValue,
+                http2_prior_knowledge: false,
+                monitoring_options: options.monitoringOptions?.rawValue,
+                host: options.hostName.awsByteCursor,
+                port: options.port,
+                initial_settings_array: nil,
```

Comment:
likewise, please get these spaced the same way they used to be spaced. for the sake of clean diffs

we'll fix the formatting in a future pass

Reply:
Thanks, fixed.